common/src/main/resources/assets/estrogen/lang/de_de.json

```
@@ -1,10 +1,16 @@
 {
   "advancement.estrogen.balls.description": "Was passiert, wenn man den Schleim aus dem Ball presst??",
   "advancement.estrogen.balls.title": "Hehe",
+  "advancement.estrogen.cookie_jar.description": "Diese Mod braucht Kekse um zu funktionieren. Bitte lege diese in das Glass.",
```

```suggestion
  "advancement.estrogen.cookie_jar.description": "Diese Mod braucht Cookies um zu funktionieren. Bitte lege diese in das Glas.",
```

Kekse would be the correct translation, and the vanilla item is called "Keks" in German. But the joke about cookies on the web gets lost, since those aren't usually translated to German. The name "Cookie" for the baked good is also commonly known in German (at least for the specific chocolate chip cookie that is in the game), so I don't think it'd cause any issues.

Also there's only one 's' in German ;)

---

That joke totally flew over my head xD, thanks for catching that